src/brasil/gov/agenda/tiles/agenda.py

```diff
@@ -179,6 +179,10 @@ def results(self):
         data = data_mgr.get()
         return data
 
+    @property
+    def date(self):
```

Comment:
this has to be, in fact, a property: refactor as `self.date = datetime.now()` in `setup()`.

Reply:
there is no ``setup()`` at tile